### DISCUSSION TITLE
Error Occurred

### DISCUSSION THREAD

**Discussion by zhi-yi-huang** [PaddlePaddle/PaddleSpatial#162] [BaiduKDDCup2022] [answered]
Do you allow larger input variables when testing? I'm confused about this.

**Comment by ramber1836** [comment, marked as answer]
What do you mean by larger input variables?

**Reply by zhi-yi-huang** [reply]
The default number of input variables are 10.  But we designed a model larger than 10.

**Reply by ramber1836** [reply]
You can use the first three columns of the input (i.e. turbine_id, day, time) as well. But no external features can be used.

**Reply by zhi-yi-huang** [reply]
So how do I use location information?

**Reply by ramber1836** [reply]
The relative locations of these 134 turbines are recorded in the file 'sdwpf_baidukddcup2022_turb_location.csv', which is within the same folder as the file 'wtbdata_245days.csv'.

**Reply by zhi-yi-huang** [reply]
I need a larger number of input to use the location information.

**Reply by ramber1836** [reply]
It is OK if you add the location information to the input dataframe as long as the output obeys the correct shape.
